src/main/webapp/recipe-display.html

```diff
@@ -70,6 +71,18 @@
           })
       }
 
+      function deletePost(recipeID) {
+        // eslint-disable-line no-unused-vars
+        const request = new Request("/api/post?recipeID=" + recipeID, { method: "DELETE" })
```

Comment:
Not a big deal, but for consistency's sake (see submit-recipe.html) it might be best to just put this directly into the fetch()

Reply:
Will do!